lemming.go

```diff
@@ -107,9 +102,13 @@ func New(lis net.Listener, targetName, zapiURL string, opts ...grpc.ServerOption
 	d.startServer()
 
 	cacheClient := gnmiServer.LocalClient()
+	yclient, err := ygnmi.NewClient(cacheClient, ygnmi.WithTarget(targetName))
+	if err != nil {
+		return nil, err
+	}
 
 	log.Infof("starting sysrib")
-	sysribServer, err := sysrib.New(sysDataplane)
+	sysribServer, err := sysrib.New(&sysrib.Dataplane{Client: yclient})
```

Comment:
Remove this parameter and in sysrib/server.go, within func (s *Server) Start, set the ygnmi client for the dataplaneAPI field.